src/Markdig/Helpers/StringSlice.cs

```diff
@@ -81,7 +83,7 @@ public StringSlice(string text, int start, int end, NewLine newLine)
         /// <summary>
         /// The text of this slice.
         /// </summary>
-        public readonly string Text;
+        public string Text;
```

Comment:
Why this change?

Reply:
Good catch! Thank you. This was accidental when backing out of changing this to a readonly property.